cas-server-support-saml/src/main/java/org/jasig/cas/support/saml/web/view/Saml10SuccessResponseView.java

```diff
@@ -96,7 +96,6 @@ protected void prepareResponse(final Response response, final Map<String, Object
      */
     private Map<String, Object> prepareSamlAttributes(final Map<String, Object> model, final Service service) {
         final Map<String, Object> authnAttributes = new HashMap<>(getAuthenticationAttributesAsMultiValuedAttributes(model));
```

Comment:
This has to be reverted. The reason attributes are wrapped is because they need to be manipulated from Map later, and the may we receive is immutable. It might be better to fix the underlying map than to change the wrapper call.